docs/installationAndConfiguration.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Installation & Configuration
+nav_order: 2
+---
+
+Install this tool using the VS Code built-in [marketplace](https://marketplace.visualstudio.com/items?itemName=mlxprs.mlxprs). Search “MarkLogic” from the Extension tab of the activity bar. Click “Install” to download and install the extension.
+
+
+
+The MarkLogic extension may be configured in the VS Code Settings. You can also change the configuration settings just for the workspace by adding the values to the `settings.json` file in the .vscode directory of your workspace.
+
+```json
```

Comment:
This comment is out-of-scope of this PR, but we'll eventually want a table that lists all the possible settings and what appropriate values are.